sirepo/package_data/static/js/raydata.js

```diff
@@ -522,7 +564,7 @@ SIREPO.app.directive('scansTable', function() {
 
             $scope.setSelectedScan = (scan) => {
                 $scope.selectedScan = scan;
-                if ($scope.selectedScan !== null) {
+                if ($scope.selectedScan !== null && $scope.selectedScan.status !== $scope.analysisStatusNone) {
```

Comment:
I think this should include pending scans too.

Reply:
18d9639913cbfbaea48046856468f9dcbb7714aa